app/controllers/carts_controller.rb

```diff
@@ -41,9 +44,10 @@ def create
   # PATCH/PUT /carts/1
   # PATCH/PUT /carts/1.json
   def update
+    #do I need the following line because I'm using load_and_authorize_resource? not sure yet
     respond_to do |format|
       if @cart.update(cart_params)
-        format.html { redirect_to @cart, notice: 'Cart was successfully updated.' }
+        format.html { redirect_to @cart, notice: 'No attributes to update.' }
```

Comment:
we shouldn't provide edit and update actions for cart as there are no attributes to update. It doesn't make sense.